packages/internal/src/ts2js.ts

```diff
@@ -48,25 +50,33 @@ export const convertTsFilesToJs = (cwd: string, files: string[]) => {
       fs.unlinkSync(path.join(cwd, f))
     }
   }
+}
 
-  if (fs.existsSync(path.join(cwd, 'api/tsconfig.json'))) {
-    fs.renameSync(
-      path.join(cwd, 'api/tsconfig.json'),
-      path.join(cwd, 'api/jsconfig.json')
-    )
-  }
-  if (fs.existsSync(path.join(cwd, 'web/tsconfig.json'))) {
-    fs.renameSync(
-      path.join(cwd, 'web/tsconfig.json'),
-      path.join(cwd, 'web/jsconfig.json')
-    )
-  }
-  if (fs.existsSync(path.join(cwd, 'scripts/tsconfig.json'))) {
-    fs.renameSync(
-      path.join(cwd, 'scripts/tsconfig.json'),
-      path.join(cwd, 'scripts/jsconfig.json')
-    )
-  }
+/**
+ * @param {string} cwd
+ */
+export function convertTsConfigsToJsConfigs(cwd = getPaths().base) {
+  const tsConfigs = [
```

Comment:
Wouldn't it be better to use a glob to search for all tsconfigs in the generated project?

Reply:
Done in https://github.com/redwoodjs/redwood/pull/4263/commits/941557b1e1eebb449c475eed04ce331c361481bb.